rules/groups_settings_rules.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Rule Keys:
+# name: The unique name describing the rule
+# mode: Depending on whether you want to allow or block in a given situation,
+#      whitelist: violates when any specified propery:value combination
+#                 doesn't match
+#      blacklist: violates when all specified propery:value combinations
+#                 match
+# mode can be whitelist or blacklist
+# group_emails: unique emails matching gsuite groups
+# settings: settings rules.  Supported settings are
+#           - whoCanAdd
+#           - whoCanJoin
+#           - whoCanViewMembership
+#           - whoCanViewGroup
+#           - whoCanInvite
+#           - allowExternalMembers
+#           - whoCanLeaveGroup
+
+rules:
```

Comment:
@ahoying @blueandgold 

What do you guys think some good sample rules to have in place would be?

Reply:
@joecheuk